README.md

```diff
@@ -1,4 +1,4 @@
-# pino-http&nbsp;&nbsp;[![Build Status](https://travis-ci.org/pinojs/pino-http.svg)](https://travis-ci.org/pinojs/pino-http)[![Coverage Status](https://coveralls.io/repos/github/pinojs/pino-http/badge.svg?branch=master)](https://coveralls.io/github/pinojs/pino-http?branch=master)
+# pino-http&nbsp;&nbsp;[![Build Status](https://img.shields.io/github/workflow/status/pinojs/pino-http/CI)](https://github.com/pinojs/pino-http/actions)[![Coverage Status](https://coveralls.io/repos/github/pinojs/pino-http/badge.svg?branch=master)](https://coveralls.io/github/pinojs/pino-http?branch=master)
```

Comment:
can you remove coveralls as well? I'm not sure it's running anymore

Reply:
Sure I can remove it. I can work on coverage if you want in another pr as well. I noticed: 
```
-----------|----------|----------|----------|----------|-------------------|
File       |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
-----------|----------|----------|----------|----------|-------------------|
All files  |    98.98 |     95.4 |      100 |    98.95 |                   |
 logger.js |    98.98 |     95.4 |      100 |    98.95 |               125 |
-----------|----------|----------|----------|----------|-------------------|
```

Reply:
Go for it!